packages/checksum/templates/README.md.hbs

```diff
@@ -0,0 +1,33 @@
+# @cumulus/checksum
```

Comment:
I like having our API documented in the README, but I kinda dislike that the README contents get hidden in this file and it's less obvious for future package maintenance where to update the README

Reply:
In addition to the "Generated automatically using `npm run build-docs`" comment at the bottom of the README.md, we could add "from `templates/README.md.hbs`?

Reply:
I think this bears a larger discussion but I don't have a great answer and it's not worth holding up this PR on